src/main/java/com/networknt/schema/ValidationMessage.java

```diff
@@ -113,17 +130,17 @@ public void setType(String type) {
         this.type = type;
     }
 
-    public static ValidationMessage of(String type, ErrorMessageType errorMessageType, String at, String... arguments) {
+    public static ValidationMessage of(String type, ErrorMessageType errorMessageType, String at, String schemaPath, String... arguments) {

```

Comment:
This is a breaking change, this should at least have been a bump of the minor version!

What's even worse, with the rest `arguments` parameter also being of the same type (`String`), any call to `ValidationMessage.of` that includes at least one argument **silently** shifts the first rest argument to become the `schemaPath` instead. No compiler warnings or anything to help detect this. Since this requires a manual search, I feel the [changelog entry](https://github.com/networknt/json-schema-validator/blob/4906d90c6f3c2dc63646bb09f0a0bdc11b6cd073/CHANGELOG.md?plain=1#L24) could have been a bit more detailed about this too.

I had some fun finding this out going from `1.0.68` → `1.0.69`, I was lucky to have a test fail on it to detect the arguments shift…